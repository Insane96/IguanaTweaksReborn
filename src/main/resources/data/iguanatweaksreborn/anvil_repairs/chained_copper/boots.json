{
	"item_to_repair": {
		"id": "iguanatweaksreborn:chained_copper_boots"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:copper_ingot"
			},
			"amount": 3
		}
	]
}