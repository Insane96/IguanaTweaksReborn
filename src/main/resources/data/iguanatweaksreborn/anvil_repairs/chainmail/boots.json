{
	"item_to_repair": {
		"id": "minecraft:chainmail_boots"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:chain"
			},
			"amount": 2
		}
	]
}