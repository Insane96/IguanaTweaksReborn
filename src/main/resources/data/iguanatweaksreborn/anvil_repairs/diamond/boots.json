{
	"item_to_repair": {
		"id": "minecraft:diamond_boots"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:diamond"
			},
			"amount": 3
		}
	]
}