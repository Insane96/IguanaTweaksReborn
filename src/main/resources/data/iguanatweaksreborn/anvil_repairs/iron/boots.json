{
	"item_to_repair": {
		"id": "minecraft:iron_boots"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:iron_ingot"
			},
			"amount": 4
		}
	]
}