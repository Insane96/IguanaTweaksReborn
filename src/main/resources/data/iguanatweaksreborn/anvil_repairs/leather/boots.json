{
	"item_to_repair": "minecraft:leather_boots",
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:leather"
			},
			"amount": 4
		},
		{
			"repair_material": {
				"id": "minecraft:rabbit_hide"
			},
			"amount": 12
		}
	]
}