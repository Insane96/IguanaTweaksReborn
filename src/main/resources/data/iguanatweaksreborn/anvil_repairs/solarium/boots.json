{
	"item_to_repair": {
		"id": "iguanatweaksreborn:solarium_boots"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:iron_ingot"
			},
			"amount": 4,
			"max_repair": 0.5
		},
		{
			"repair_material": {
				"id": "iguanatweaksreborn:solarium_ball"
			},
			"amount": 2
		}
	]
}