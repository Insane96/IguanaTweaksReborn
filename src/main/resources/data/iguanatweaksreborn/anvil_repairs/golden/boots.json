{
	"item_to_repair": {
		"id": "minecraft:golden_boots"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:gold_ingot"
			},
			"amount": 4
		}
	]
}